this_is_a_test.py

```diff
@@ -1,2 +1,5 @@
 # My passwords
 a_password = "my_sp3c1f1c_pa$$w0rd_trav1s_ll0yd_1"
+pwd = 'KB7lZhEr4j'
+ano_pwd = 'HbDlGPTSpJ'
+a_passwd = 'A8xuCpynj2'
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 906fd9e0-b64d-4dde-8112-528f089708b7 -->